src/features/student/declaredPrograms/components/interactions/formFields/DeclaredProgramTitleFormField/DeclaredProgramTitleFormField.vue

```diff
@@ -0,0 +1,37 @@
+<script setup lang="ts">
+import type { AddDeclaredProgramForm } from '@/features/student/declaredPrograms/types/forms.types'
+import DeclaredProgramTitleInput from '@/features/student/declaredPrograms/components/interactions/inputs/DeclaredProgramTitleInput/DeclaredProgramTitleInput.vue'
+
+import { DECLARED_PROGRAM_TITLE_MAX_LENGTH } from '@/features/student/declaredPrograms/config'
```

Comment:
```suggestion
import DeclaredProgramTitleInput from '@/features/student/declaredPrograms/components/interactions/inputs/DeclaredProgramTitleInput/DeclaredProgramTitleInput.vue'
import { DECLARED_PROGRAM_TITLE_MAX_LENGTH } from '@/features/student/declaredPrograms/config'
```

Un saut de ligne en trop